src/lib/konnectors.js

```diff
@@ -65,6 +65,14 @@ export function isKonnectorKnownError(error) {
   return error && error.type && Object.keys(ERROR_TYPES).includes(error.type)
 }
 
+export function isKonnectorMaintenanceError(error) {
```

Comment:
This code is perfectly correct, but after discussing with our Product Owner, I think we will keep the logic of making a difference between `errors` and `userErrors`.